multi-node/aws/README.md

```diff
@@ -24,6 +24,32 @@ $ aws s3 cp --recursive --acl=public-read artifacts/ s3://<bucket>/
 
 Then, simply create a cluster using `artifactURL: https://<bucket>.s3.amazonaws.com`.
 
+### Accessing the Kubernetes Cluster Locally
+
+```
+$ ssh -f -nNT -L 8080:127.0.0.1:8080 -i <private-key> core@<cluster-external-dns-name>
```

Comment:
I'd prefer that we document using `kubectl proxy` as the method for tunneling to apiserver. The above will also work, but the contract we are maintaining is that "apiserver is accessible via config options in a kubeconfig". With that contract, `kubectl proxy` should be able to handle tunnel/auth for us.